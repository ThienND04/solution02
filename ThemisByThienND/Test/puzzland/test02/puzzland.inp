7
14 67
I S A D F E M X B P K U T V 
A T
K U
F X
S D
F B
I F
S K
F T
D E
D K
D F
D V
M K
M U
K T
D M
B K
P K
P V
E M
F P
F E
I B
E X
D T
E P
E V
A X
S P
A D
B U
F V
B P
B T
S E
E K
S V
I A
P U
I P
A K
M P
X T
I S
S U
I M
X P
D B
I U
E B
X B
A U
M B
B V
A F
E U
X U
M X
A M
T V
F K
I T
M V
K V
M T
I D
A P
12 45
Y H V O E T W J I U M G 
O M
Y E
U M
Y U
T W
M G
Y M
J U
Y O
H T
Y G
E M
Y W
H W
Y J
U G
H J
W I
V M
V E
J M
V O
O I
V W
V T
Y I
E W
E G
V J
H V
T I
I U
E I
O E
T J
H M
O U
V G
W G
T U
J I
H O
H G
T G
Y T
13 73
B A G Z M L Y H I S D K C 
Z S
S C
G I
Y D
G Z
H C
A D
Z D
M L
K C
G Y
I S
Z H
A M
A I
M I
Y H
B L
L I
D C
A Y
A S
I C
G L
M D
M S
B Z
H K
A K
G S
I D
L Y
L S
S K
Z I
G C
M C
B G
B I
I K
G M
Y K
B H
H I
H S
B M
Y I
S D
M Y
A L
B D
M H
A C
Y C
B Y
D K
Y S
B K
L K
A Z
H D
L D
G H
B A
Z K
L C
A H
Z L
B C
Z C
L H
G D
Z Y
15 103
D L X O E Q W I T K A C F Y V 
T K
X I
D Y
L A
E F
O K
E K
D L
Y V
T C
D C
D X
L W
L E
E A
X Y
E I
K F
C V
O A
W F
K C
L V
Q C
L O
F Y
C Y
I C
W I
X T
O F
L F
D V
A C
D E
Q K
E C
A Y
L Q
E V
D I
F V
T F
E W
K V
W C
O V
Q Y
T V
C F
I Y
O Y
L I
X E
E Q
D A
A V
O E
X V
L K
X W
O Q
W T
D Q
W A
I F
O C
X O
L T
Q F
X A
L X
I K
T A
X F
X C
I T
K Y
Q A
E T
L Y
E Y
W K
O T
O W
Q I
D K
I A
W Y
D W
X Q
L C
Q V
Q W
D F
D T
D O
O I
Q T
W V
K A
A F
X K
14 85
S G I P H O A F V Y T L E U 
G L
O L
S T
V Y
G H
H Y
A E
I A
S Y
S L
O Y
S F
A Y
F Y
G I
P T
G P
I O
L E
O E
G A
G T
G Y
O A
H T
Y U
O F
P L
T E
Y E
P Y
H A
S G
I E
S P
H O
P A
H E
A V
G U
P H
I F
I H
S I
E U
G O
I Y
Y T
L U
I L
A T
H F
A U
F T
I V
G E
S O
O U
T L
P U
I P
S U
F U
V E
A L
V T
H L
Y L
S H
P F
V L
G F
P E
S E
S V
A F
H U
G V
I T
F V
F E
T U
O V
S A
V U
15 62
X H M O B T Z J F Y I D W V R 
H D
H B
T Z
M Z
Z J
M Y
J D
T V
X W
H J
M I
X M
X I
Z W
Z V
X D
T W
O Y
O F
F D
X V
X R
B Y
B I
B Z
F R
B F
Y W
J R
I R
H Y
M D
B V
Z Y
O Z
O V
H Z
J V
X B
V R
J W
M V
I D
Z F
M F
Y I
H W
W V
J F
D R
T F
M T
T D
M B
M W
Z I
I W
O R
M J
O I
J I
B W
13 77
Z C U E G X W Q B S I H N 
Q N
E H
Q B
Z Q
C S
X S
S H
Z G
Q H
W N
E I
C E
Z X
U X
W I
X I
G N
G X
B H
W H
U I
Q I
S I
E S
C B
Z W
C G
W S
B N
B S
U B
G I
Z E
W B
Q S
X Q
Z B
U Q
U S
E G
B I
W Q
Z I
E B
C X
C N
E Q
Z U
I H
G H
U G
C W
H N
G S
Z N
S N
Z S
C U
X B
I N
U E
Z H
C Q
E W
U W
X H
Z C
G W
E X
C H
X N
C I
E N
X W
G Q
G B
U H
