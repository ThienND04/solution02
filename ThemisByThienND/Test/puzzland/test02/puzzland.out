IADBEFKMPSUXTV
YEIJHMOUTVWG
BADGHIKLMSYZC
DACEFIKLOQTWXYV
SAEFGHILOVTPYU
XBFDHJIMTVWYZOR
ZBCEGHIQSUWXN
